docs/demos/basic/index.html

```diff
@@ -12,7 +12,7 @@
     <script src="../../panelsnap.js" defer></script>
     <script>
       document.addEventListener("DOMContentLoaded", function () {
-        new PanelSnap();
+        new PanelSnap({timeout: 10});
```

Comment:
I'd like the basic demo to be a clean as possible. Zero config which shows we have sane defaults. Can we either create a new demo or just revert this one line?